README.md

```diff
@@ -61,9 +61,12 @@ All these endpoints accepts any combination of the following parameters:
 * `issue_date_end` (exclusive)
 * `month`
 * `subquota_id`
-* `suspicions` (_boolean_, basically any value parses to `True`)
+* `suspicions` (_boolean_, `1` parses to `True`, `0` to `False`)
 * `year`
-* `order_by`: `issue_date` (default) or `probability` (both descending)
+* `order_by`: `issue_date` (default) or `probability`
+(both descending)
```

Comment:
Not necessarily a typo because in `.md` this line break has no effect — but surely it wasn't intentional. Removed in 9723951.